meson.build

```diff
@@ -148,8 +148,6 @@ else
   have_networkmanager = false
 endif
 
-payg_dep = dependency('eos-payg-1', version: payg_req, required: false)
```

Comment:
Sorry, I didn't notice this before.. I think you could also remove the `payg_req` variable defined somewhere above.